mosquitto/Dockerfile

```diff
@@ -1,16 +1,16 @@
-ARG BUILD_FROM
+ARG BUILD_FROM=ghcr.io/home-assistant/amd64-base:3.13
 FROM $BUILD_FROM
 
+# Set shell
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
```

Comment:
only comment: We can also set bash as the default on the base image. I think currently it's ash on alpine

Reply:
Removing it